lhotse/audio/recording.py

```diff
@@ -294,19 +295,21 @@ def _aslist(x):
             return x
 
         # Case #1: no opts specified, read audio without decoding and move it in memory.
-        if all(opt is None for opt in (channels, offset, duration)) or (
+        if format is None and (all(opt is None for opt in (channels, offset, duration)) or (
```

Comment:
`if all(opt is None for opt in (channels, offset, duration, format))` would be clearer